modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -68,7 +68,11 @@ object types {
   case class RewardsConfig(
     programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
     rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch,
-    oneTimeRewards: List[OneTimeReward] = List.empty
+    oneTimeRewards: List[OneTimeReward] = List(
+      // Transferring current balance of 43,430,294.88479231 from DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS
```

Comment:
If we want to leave this comment I would be more precise and say "current balance as of <ordinal> where the last balance update happened" or something similar. In the future "current" will not sound right.